docs/sources/setup-grafana/configure-security/configure-authentication/jwt/index.md

```diff
@@ -94,7 +94,7 @@ you can use JWT authentication to authenticate the iframe.
 
 {{% admonition type="note" %}}
 For Grafana Cloud, or scenarios where verifying viewer identity is not required,
-embed [public dashboards]({{< relref "../../../../dashboards/dashboard-public" >}}).
+embed [public dashboards](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/dashboards/share-dashboards-panels/shared-dashboards/).
```

Comment:
```suggestion
embed [shared dashboards](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/dashboards/share-dashboards-panels/shared-dashboards/).
```

Reply:
Or are we keeping the term 'Public Dashboards'? There are a few other places in the document that still has this naming but if we are renaming everything, to 'Shared Dashboards" would it be confusing to use the term 'Public Dashboards'?